autorosters/autorosters_main.py

```diff
@@ -39,15 +39,16 @@ def run(self):
     def get_tabs(self):
         page = self.site.client.pages[self.overview_page]
         page_text = page.text()
-        self.tabs = re.search(r'{{Tabs:(.*?)}}', page_text).group(1) or ""
+        tabs = re.search(r'{{Tabs:(.*?)}}', page_text)
+        self.tabs = tabs[1] if tabs else ""
```

Comment:
this is gonna result as a blank line in the case that there are no tabs on the overview page.

(1) maybe do a special handling in this case (else None and then later if tabs are None don't concatenate them along with the newline)
(2) if there are no tabs, that's really bad, so a warning should be printed to the user. You can do something like self.warnings = [] and then self.warnings.append("There are no tabs on the overview page!") and then after the process is done, print all of the warnings back to the user from the main execution function